prometheus_client/gc_collector.py

```diff
@@ -23,28 +29,17 @@ def __init__(self, registry=REGISTRY, gc=gc):
         if not hasattr(gc, 'callbacks'):
             return
 
-        collected = Histogram(
-            'python_gc_collected_objects',
-            'Objects collected during gc',
-            ['generation'],
-            buckets=[500, 1000, 5000, 10000, 50000],
-            registry=registry
-        )
+        self._buckets = [{
+            self._LATENCY: defaultdict(lambda: 0.0),
```

Comment:
Are default dicts threadsafe? Can they do allocs?

Reply:
yes defaultdicts threadsafe but we should have locks for incrementing : (

Reply:
I forget about it